azure/windows-webapp/windows-webapp/examples/basic/main.tf

```diff
@@ -0,0 +1,42 @@
+provider "azurerm" {
+  features {
```

Comment:
Are these additional features a requirement to use the module? If so, is it possible to include them within the provider definitions within the module?

Reply:
This is in the example to prevent resources from hanging around / force-delete.